### PR TITLE
TwitterTrack: Component wrapping twitter universal lib

### PR DESCRIPTION
**Pull request by Attamusc**

### CODE REVIEW

File: test/lib/karma.conf.js [hunk 1]
```diff
@@ -52,6 +52,10 @@ module.exports = function(config) {
       dir: 'test/coverage/'
     },
 
+    mochaReporter: {
+      ignoreSkipped: true
```
Comment (weikinhuang) [inline test/lib/karma.conf.js]: seems random

**Comment by weikinhuang** [comment]
:+1:

File: test/lib/karma.conf.js (hunk 1 above)
Comment (Attamusc) [inline test/lib/karma.conf.js]: BeFF shows ALL TESTS even when focused which makes TDD super hard to do. All the other repos already do this so just adding it while I remember it annoys me.

**Comment by weikinhuang** [comment]
:+1:

**Comment by designjockey** [comment]
👍

**Pull request merged**